src/app/content/selectors.ts

```diff
@@ -82,3 +82,8 @@ export const prevNextPage = createSelector(
     ? prevNextBookPage(selectedBook, selectedPage.id)
     : null
 );
+
+export const enableStudyGuides = createSelector(
+  localState,
+  (state) => state.enableStudyGuides
```

Comment:
I don't think this enables anything, maybe change it to something like `studyGuidesEnabled`

Reply:
I was following the previous convention, but since now it is the only feature flag it should be good to update it